openreview/conference/templates/paper_recruitment_process.py

```diff
@@ -56,9 +59,20 @@ def process(client, note, invitation):
 
 OpenReview Team'''
             response = client.post_message(subject, [edge.tail], message)
+
+            ## Send email to inviter
+            subject=f'[{SHORT_PHRASE}] {REVIEWER_NAME} {preferred_name} accepted to review paper {submission.number}, conflict detection pending'
+            message =f'''Hi {{{{fullname}}}},
+The {REVIEWER_NAME} {preferred_name} that you invited to review paper {submission.number} has accepted the invitation to review paper {submission.number}.
```

Comment:
@celestemartinez please check the wording here

Reply:
Seems redundant to repeat the submission number twice. Maybe something like this:

> The reviewer {} that you invited to review paper {} has accepted the invitation.
> 
> Confirmation of the assignment is pending until the invited reviewer creates a profile in OpenReview and no conflicts of interest are detected.